rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Same here,  the Getters and Setters for @book and @person instance variables should be present

```ruby
  attr_accessor :date, :book, :person
```